Content.Client/Shipyard/UI/ShipyardConsoleMenu.xaml.cs

```diff
@@ -74,6 +74,8 @@ public void PopulateProducts(ShipyardConsoleUiKey uiKey)
             ShipyardConsoleUiKey.Security => "Security",
             ShipyardConsoleUiKey.BlackMarket => "BlackMarket",
             ShipyardConsoleUiKey.Expedition => "Expedition",
+            ShipyardConsoleUiKey.SaleOnly => "SaleOnly",
+            ShipyardConsoleUiKey.Specialty => "Specialty",
```

Comment:
needs to be more unique and descriptive, 'specialty' is too generic

Reply:
"Eccentric" maybe? Or just simply "Tinnias" but that's boring